pyface/ui_traits.py

```diff
@@ -19,6 +19,10 @@
 
 from traits.api import ABCHasStrictTraits, Enum, Range, TraitError, TraitType
 from traits.trait_base import get_resource_path
+try:
+    from traits.trait_handlers import CALLABLE_AND_ARGS_DEFAULT_VALUE
```

Comment:
Could we insist on minimum Traits version of 5.0.0, to avoid this `try` / `except`? Anyone upgrading to the latest Traits UI should really be also updating to the latest Traits at the same time.

Reply:
We could, but I was hoping to pull this into a bugfix release and it seems wrong to insist on a dependency major version change in a bugfix release